src/components/portfolios/PortfolioCard.jsx

```diff
@@ -0,0 +1,43 @@
+import Card from "../common/atoms/Card";
+import { comma } from "../../utils/convert";
+import SquarePhoto from "../common/atoms/SquarePhoto";
+
+const PortfolioCard = ({ portfolio }) => {
+  return (
+    <Card to={`/portfolios/${portfolio.id}`} className="portfolio-card">
+      <SquarePhoto
+        src={`${portfolio.image.substring(1, portfolio.image.length)}`}
```

Comment:
템플릿 리터럴을 사용해주셨는데 사실 템플릿 리터럴을 쓰지 않고 그냥 substring 메소드의 결과물을 바로 바인딩하셔도 괜찮습니다!
템플릿 리터럴을 쓰면 이 내부에 여러개의 변수가 쓰였다거나 다른 문자와의 결합되는 것을 기대하고 코드를 보게 되니까 오히려 헷갈리게 되는 것 같아요!
```suggestion
        src={portfolio.image.substring(1, portfolio.image.length)}
```

Reply:
수정했습니다!